### DISCUSSION TITLE
Table Row Event on Vue 3/Ant-design -vue 2.2.8

### DISCUSSION THREAD

**Discussion by gabriellatavares** [vueComponent/ant-design-vue#4853] [Q&A] [answered]
Hello! First of all, thanks for the amazing project! I was wondering if it's possible to make a event on click in the row of table from ant-desing-vue. 

I have a sortable table and I need to open a model on double click of the **table row.**  I'm using ant-design-vue  2.2.8 with Vue 3. 

Does anyone knows how? If it's not possible, do you guys recommend an alternative? 

Thank you so much.

**Comment by zkwolf** [comment, marked as answer]
You can use customRow to bind click event to row.
